app/models/settings/smtp.rb

```diff
@@ -0,0 +1,18 @@
+module Settings
+  class SMTP < RailsSettings::Base
+    self.table_name = :settings_smtps
+
+    OPTIONS = %i[address port authentication user_name password domain].freeze
+
+    # The configuration is cached, change this if you want to force update
+    # the cache, or call Settings::Smtp.clear_cache
+    cache_prefix { "v1" }
+
+    field :address, type: :string, default: ApplicationConfig["SMTP_ADDRESS"]
```

Comment:
Total nitpick but in all the other settings files we kept the settings sorted alphabetically.

Reply:
sorted!